src/App/App.js

```diff
@@ -1,8 +1,12 @@
 import React, { Component } from 'react';
-import Studios from '../Studios/Studios.js';
+import Header from '../Header/Header.js';
+import Carousel from '../Carousel/Carousel.js';
+import TypeCard from '../TypeCard/TypeCard.js';
 import search from '../Search/SearchFunction.js';
 import SearchBar from '../Search/SearchBar.js';
-import Type from '../Type/Type.js';
+import Studios from '../Studios/Studios.js';
```

Comment:
Do we need to import `search` and `TypeCard` here? I'm not seeing them being utilized (or maybe I'm missing it?).